SPXIronCondors.py

```diff
@@ -0,0 +1,238 @@
+'''
+SPX 0 DTE backtesting: Every day, since SPX now has Tuesday and Thursday expirations
+
+SPX 4 DTE backtesting: Monday -> Friday
+'''
+import pandas as pd
+import numpy as np
+from scipy.stats import norm
+from scipy.optimize import minimize
+from util.Enums import Ticker, Columns
+from OptionPrice import Constants
+
+class SPX0DTE:
+    '''
+    Let S[n] be today's closing price on SPX and S[n+1] be tomorrow's closing price.
+
+    Let p be the probability of profit (pop) threshold on one side of the iron condor.
```

Comment:
```suggestion
    Let p be the probability of profit (PoP) threshold on one side of the iron condor.
```

Nit: I think you were using PoP later, but this is whatever.

Reply:
I probably prefer P(profit), but this is fine

Reply:
PoP is the standard acronym